questions/css-box-model.md

```diff
@@ -0,0 +1,31 @@
+### Your question goes here. Keep it straightforward.
+
+Describe the layout of the CSS Box Model and briefly describe each component.
+
+#### Answer
+
+<!-- Your answer goes here. -->
+
+*Content*: The inner-most part of the box filled with text or an image or some type of content</br>
```

Comment:
Markdown does not use break tags. They can be removed. Double new lines is what is used instead